ecs/cloudformation.go

```diff
@@ -293,10 +184,46 @@ func (b *ecsAPIService) convert(project *types.Project) (*cloudformation.Templat
 			TaskDefinition:     cloudformation.Ref(normalizeResourceName(taskDefinition)),
 		}
 	}
-	return template, networks, nil
+	return template, nil
+}
+
+func (b *ecsAPIService) createIngress(service types.ServiceConfig, net string, port types.ServicePortConfig, template *cloudformation.Template) {
+	protocol := strings.ToUpper(port.Protocol)
+	if protocol == "" {
+		protocol = "-1"
```

Comment:
some strange AWS required defaults ?? :) 
a constant with an explicit name might make this less obscure